openid-federation-1_0.xml

```diff
@@ -7386,6 +7386,21 @@ HTTP/1.1 302 Found
           as mandated in <xref target="fetching-es"/>.
 	</t>
       </section>
+        <section anchor="anchor_resolver" title="Trust Anchors and Resolvers go together">
+            <t>
+            If you plan to have one resolver in your federation you should have the same entity being
+            both the Trust Anchor and the Resolver. If you do that users of the resolver will not have to
+            collect and evaluate a trust chains for the Resolver since you must trust the Trust Anchor and by
+            combining the two services the Resolver will be implicitly trusted since the Trust Anchor is trusted.
+            </t>
+    </section>
+        <section anchor="one_service" title="One entity one service">
+            <t>
+            Apart from letting an entity provide both the Trust Anchor and Resolver services there is a good reason for
+            letting each entity only do one thing. The reason is that, later in time, it will be much easier to share specific services
```

Comment:
```suggestion
            Apart from letting an entity provide both the Trust Anchor and Resolver services, there is a good reason for
            having each entity only do one thing. The reason is that, later in time, it will be much easier to share specific services
```